packages/utils/types/feed.ts

```diff
@@ -135,3 +136,27 @@ export type ReplyToType = {
   yOffsetValue?: number;
   parentId?: string;
 };
+
+export const parseSocialFeedMetadata = (
+  category: PostCategory,
+  metadata: string,
+) => {
+  switch (category) {
+    case PostCategory.Video:
+      return (
+        zodTryParseJSON(ZodSocialFeedPostMetadata, metadata) ||
+        zodTryParseJSON(ZodSocialFeedVideoMetadata, metadata)
+      );
+    case PostCategory.Article:
+      return (
+        zodTryParseJSON(ZodSocialFeedPostMetadata, metadata) ||
+        zodTryParseJSON(ZodSocialFeedArticleMetadata, metadata)
+      );
```

Comment:
```suggestion
      return (
        zodTryParseJSON(ZodSocialFeedArticleMetadata, metadata)
      );
```

Reply:
Resolved it here: https://github.com/TERITORI/teritori-dapp/pull/1400/commits/0b9046c2de0b394297ca278952dff0b02b399731